exercises/practice/difference-of-squares/difference-of-squares.spec.js

```diff
@@ -0,0 +1,74 @@
+import { compileWat } from './compile-wat';
+
+let wasmModule;
+let currentInstance;
+
+beforeAll(async () => {
+  try {
+    const {buffer} = await compileWat("difference-of-squares.wat");
+    wasmModule = await WebAssembly.compile(buffer);
+  } catch (err) {
+    console.log(`Error compiling *.wat: ${err}`);
+  }
+});
+
+
+describe('difference-of-squares', () => {
+  beforeEach(async (done) => {
+    currentInstance = null;
+
+    if (!wasmModule) {
+      done();
+      return;
+    }
+    try {
+      currentInstance = await WebAssembly.instantiate(wasmModule);
+    } catch (err) {
+      console.log(`Error instantiating WebAssembly module: ${err}`);
+    }
+    done();
+  });
+
+
+  describe('Square the sum of the numbers up to the given number', () => {
+    test('square of sum 1', () => {
+      expect(currentInstance.exports.squareOfSum(1)).toBe(1);
+    });
+
+    test('square of sum 5', () => {
```

Comment:
Did you purposefully not use `xtest` this time?

```suggestion
    xtest('square of sum 5', () => {
```